packages/server/src/api/index.ts

```diff
@@ -2594,23 +2594,14 @@ router.get('/user/article-activities/:size/:cursor', async (req, res) => {
 router.get('/user/article-stories', async (req, res) => {
     try {
       const prisma = PrismaClientSingleton.prisma;
-      const { size, cursor } = req.query; 
-  
-      // Define the default page size and cursor if not provided
-      const pageSize = size ? size : 10; // Default to 10 items per page
-      const cursorId = cursor ? cursor : 0; 
   
       const articleStories = await prisma.articleStoryDistribution.findMany({
         where: {
           userId: res.locals.userId, // Filter by the logged-in user's ID
```

Comment:
Nice work but `res.locals.userId` is string ( UUID ) and `prisma userId` is number hence it will not find the correct stories.

Reply:
Try to go through user in that case res.locals.email is user unique identifier that is available to your route. Hope it makes sense

Reply:
Need help ask it here or in discord . Thanks

Reply:
ok makes sense . thanks.

Reply:
@patil-gau Yes exactly, and sorry for the late reply on this issue, I was stuck on a bug.